examples/ui/main.go

```diff
@@ -30,9 +31,29 @@ func ExampleHandler1(w http.ResponseWriter, _ *http.Request) {
 	fmt.Fprint(w, htmlPage)
 }
 
+func AccessibilityExclusionHandler(w http.ResponseWriter, _ *http.Request) {
+	htmlPage := `
+		<!doctype html>
+		<html lang=en>
+			<head>
+				<meta charset=utf-8>
+				<title>blah</title>
+			</head>
+			<body>
+				<p class='example-paragraph'>Example web page</p>
+				<label for="test">This is a test label</label>
+				<input id="test" class='example-input' value='test value'>
+				<!-- This is an accessibility failure -->
+				<image src="/example.png" />
```

Comment:
Minor and won't make a difference here, but this should be img.
but having a play around using "| image-alt |" in the exclusion works for both img and image!